pkg/interface/src/views/landscape/components/Home/Post/PostThread.tsx

```diff
@@ -0,0 +1,124 @@
+import { Box, Col, Text } from '@tlon/indigo-react';
+import bigInt from 'big-integer';
+import React, {
+  useEffect
+} from 'react';
+import { resourceFromPath } from '~/logic/lib/group';
+import { Loading } from '~/views/components/Loading';
+import useGraphState from '~/logic/state/graph';
+import PostFlatFeed from './PostFlatFeed';
+import PostItem from './PostItem/PostItem';
+
+
+export default function PostThread(props) {
+  const {
+    baseUrl,
+    api,
+    history,
+    association,
+    graphPath,
+    group,
+    vip,
+    pendingSize
+  } = props;
+
+  const graphResource =
+    graphPath ? resourceFromPath(graphPath) : resourceFromPath('/ship/~zod/null');
+
+  const locationUrl = props.locationUrl.replace(`${baseUrl}/feed/thread`, '');
```

Comment:
It's not a blocker, given that it's just matching the other Post code, but I'd like to voice how much i dislike this pattern. We should use `react-router-dom` instead, which would allow us to only construct the bigints when they change, rather than every render.